debug_toolbar/static/debug_toolbar/css/toolbar.css

```diff
@@ -236,6 +236,24 @@
 #djDebug .djDebugPanelContent .djdt-loader {
     display: block;
     margin: 80px auto;
+    border: 6px solid white;
+    border-radius: 50%;
+    border-top: 6px solid #ffe761;
+    width: 38px;
+    height: 38px;
+    -webkit-animation: spin 2s linear infinite; /* Safari */
+    animation: spin 2s linear infinite;
+}
+
+/* Safari */
+@-webkit-keyframes spin {
```

Comment:
According to MDN, Safari supports `@keyframes`, so I think we should drop this vendor prefix:

https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes

We made some efforts in the past to drop vendor prefixes: 22dbea38dd4d52ea2eb8783a2c8c8a5c8a0442a1

Worst case for older browsers, this should degrade gracefully so I don't think it will be a major issue for them.

Reply:
Sure thing, I removed them.